### DISCUSSION TITLE
Donation?

### DISCUSSION THREAD

**Discussion by jfdid** [TomBursch/kitchenowl#57] [General]
Tom, this is exactly what I've been looking for in a Grocery app.  FOSS/Self Hosted and has categories for the shopping list(most important for us - as we use it to walk through the store in proper order).  
Thank you very much for continuing to work on this.  I haven't done much on the recipe side of things but the shopping side is fantastic.

IOS official app(in App Store, rather than Flight Path) is the only thing we're waiting on now.  I'd be happy to send over a few dollars to support the project but couldn't find a link to donate?

Thanks again, and please don't let this project fade away!

**Comment by TomBursch** [comment]
Thank you so much for the kind words! I'm happy that this project helps other people. I'm not sure if I want to accept donations at the moment as I'm unaware of the legal implications and some other reasons. I'll think about it :)

**Comment by TomBursch** [comment]
Completed with #170
